src/gui/PrintInitDialog.cc

```diff
@@ -26,9 +26,9 @@
 
 #include <QString>
 
-#include "Settings.h"
-#include "PrintService.h"
-#include "PrintInitDialog.h"
+#include "gui/Settings.h"
+#include "gui/PrintService.h"
+#include "gui/PrintInitDialog.h"
 #include "QSettingsCached.h"
```

Comment:
gui/QSettingsCached.h

Reply:
I wonder why that was not picked up properly by the script before. Fixed.